### DISCUSSION TITLE
How to troubleshooting not working scheduled rescan via cron job?

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6233] [Q&A]
How can I check, in the container logs, or wherever, if the implemented cron job was really running? And if, why it does not rescan the libraries?

I enabled rescan in the GUI, the cron expression is
0 */1 * * *
but libraries are not scanned, new assets are not added. Only when I manually trigger the rescan via GUI as a specific user, rescan happens.

I need to solve this issue, to be able to use immich for multiple users.

I am ready to learn, how to use the API to trigger a rescan. I am developer, but with very limited knowledge about Unix or Linux.

But I really need to solve this rescan issue to be able to use immich. Asking normal immich users to rescan their libraries from time to to time is not a solution.

related to #6228

**Comment by bo0tzz** [comment]
The job is broken due to a bug, it'll be fixed in the next release.
